modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/SlaCard/index.js

```diff
@@ -0,0 +1,162 @@
+					slaDateEnd: slaNewDate(project.slaExpiredEndDate),
+					slaDateStart: slaNewDate(project.slaExpiredStartDate),
+					slaLabel: SLA_NAMES.expired.toUpperCase(),
+					slaTitle: project.slaExpired?.split(' ')[0],
+				},
+				future: {
+					slaDateEnd: slaNewDate(project.slaFutureEndDate),
+					slaDateStart: slaNewDate(project.slaFutureStartDate),
+					slaLabel: SLA_NAMES.future.toUpperCase(),
+					slaTitle: project.slaFuture?.split(' ')[0],
+				},
+			};
+
+			if (
+				slaRawData.current.slaTitle === slaRawData.expired.slaTitle &&
+				slaRawData.current.slaTitle === slaRawData.future.slaTitle
+			) {
+				slaRawData.current.slaDateStart =
+					slaRawData.expired.slaDateStart;
+				slaRawData.current.slaDateEnd = slaRawData.future.slaDateEnd;
+				slaFiltedData.push(slaRawData.current);
+			} else if (
+				slaRawData.current.slaTitle === slaRawData.expired.slaTitle
+			) {
+				slaRawData.current.slaDateStart =
+					slaRawData.expired.slaDateStart;
+				slaFiltedData.push(slaRawData.current);
+				slaFiltedData.push(slaRawData.future);
+			} else if (
+				slaRawData.current.slaTitle === slaRawData.future.slaTitle
+			) {
+				slaRawData.current.slaDateEnd = slaRawData.future.slaDateEnd;
+				slaFiltedData.push(slaRawData.current);
+				slaFiltedData.push(slaRawData.expired);
+			} else {
+				slaFiltedData.push(slaRawData.current);
+				slaFiltedData.push(slaRawData.expired);
+				slaFiltedData.push(slaRawData.future);
+			}
+
+			setSlaData(slaFiltedData.filter((sla) => sla.slaTitle));
+
+			if (!slaSelected) {
+				const slaSelectedCard = slaFiltedData[0]?.slaLabel;
+				setSlaSelected(slaSelectedCard);
+			}
+		}
+	}, [project, slaSelected]);
+
+	if (slaData) {
+		SLA_DATA.firstData = slaData[0];
+		SLA_DATA.secondData = slaData[1];
+		SLA_DATA.thirdData = slaData[2];
+	}
+
+	const handleSlaCardClick = () => {
+		if (slaSelected === slaData[slaData.length - 1]?.slaLabel) {
+			setSlaSelected(SLA_DATA.firstData.slaLabel);
+		} else if (slaSelected === SLA_DATA.firstData.slaLabel) {
+			setSlaSelected(SLA_DATA.secondData.slaLabel);
+		} else {
+			setSlaSelected(SLA_DATA.thirdData.slaLabel);
+		}
+	};
+
+	return (
+		<div className="cp-sla-container position-absolute">
+			<h5 className="mb-4">Support Level</h5>
+
+			{slaData ? (
+				<div>
+					<div
+						className={classNames({
+							'ml-2': slaData.length > 1,
+						})}
+					>
+						<div
+							className={classNames(
+								'align-items-center d-flex cp-sla-card-holder',
+								{
+									'cp-sla-multiple-card ml-2':
+										slaData.length > 1,
+								}
+							)}
+						>
+							{slaData &&
+								slaData.map((sla) => (
+									<SlaCardLayout
+										key={sla.slaTitle}
+										slaDateEnd={sla.slaDateEnd}
+										slaDateStart={sla.slaDateStart}
+										slaLabel={sla.slaLabel}
+										slaSelected={slaSelected}
+										slaTitle={sla.slaTitle}
+									/>
+								))}
+						</div>
+					</div>
+
+					{slaData.length > 1 && (
```

Comment:
```suggestion
					{!!slaData.length && (
```

Reply:
@jairmedeiros I think this > 1 is intencional, when I was reviewing Joana's code, she explained that to me.

but, if it's not, @Lucas2691 please make the suggestions from @jairmedeiros 

The same from the others comment

Reply:
I guess this Jair suggestions don't fit to me, because I need more than one items one array to build multiples cards. Not any items inside array.